.github/workflows/test_helper.yml

```diff
@@ -20,6 +20,16 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
+      - name: Load Secrets
```

Comment:
dbt uses a db connection to compile, and dbt sqlfluff templater compiles sql files before linting so this is needed